scrapy/templates/project/module/middlewares.py.tmpl

```diff
@@ -6,51 +6,8 @@
 # http://doc.scrapy.org/en/latest/topics/spider-middleware.html
 
 from scrapy import signals
+from scrapy.spidermiddlewares import SpiderMiddleware
 
 
-class ${ProjectName}SpiderMiddleware(object):
-    # Not all methods need to be defined. If a method is not defined,
-    # scrapy acts as if the spider middleware does not modify the
-    # passed objects.
-
-    @classmethod
-    def from_crawler(cls, crawler):
-        # This method is used by Scrapy to create your spiders.
-        s = cls()
-        crawler.signals.connect(s.spider_opened, signal=signals.spider_opened)
```

Comment:
I tried that, but different spider middleware can have different `__init__` arguments. So I believe that would only make it more restrictive on how to create your own spider middleware